### DISCUSSION TITLE
EPOS function support

### DISCUSSION THREAD

**Discussion by ripvega** [Thomas55555/husqvarna_automower#337] [Ideas]
What a splendid integration this is. Thank you for taking the time.

I have two 550 EPOS mowers and would like to see support for the Maintenance points and cutting areas.

The Husqvarna app doesn't currently allow the scheduling of maintenance points so that, for example, every Thursday at 06:00 the mowers both go to a point and await cleaning and maintenance.

Additionally, being able to select an alternate cutting area with HA would be excellent.

I don't currently have the skills to contribute much, but can certainly test it!

TIA

**Comment by Thomas55555** [comment]
As far as I know, the Husqvarna Open API doesn't support this functions. So it can't be implemented, yet.
